### DISCUSSION TITLE
Using an installation of Linux on an external HDD(or USB Drive) as a chroot?

### DISCUSSION THREAD

**Discussion by sanrobin** [dnschneid/crouton#4617] [General]
Just out of curiosity, Since I have an installation of Ubuntu Focal on my external HDD, Can I use is as a chroot? Will it work?

**Comment by ghost** [comment]
I wouldn't think that your external installation would work, although you could attempt to use the `chroot` command on the mount point, The problems I see would be that they may not use the same architecture and that X server would probably give you some issues. You could also take a look at installing Crouton to an external drive, see [this](https://github.com/dnschneid/crouton/wiki/How-To-Install-To-External-Drive) wiki page.

**Comment by wehkah** [comment]
Hi!

I am unsure, if I understand the wiki page correctly. Can Crouton install Linux completely to a USB drive, so that Linux can be run from that USB drive without any changes to the internal ChromeOS drive? 

This would be a great way to test ones Chromebook before taking any risks.

**Comment by DennisLfromGA** [comment]
@wehkah,

You can install crouton chroots to external media but, being a chroot, it still depends on ChromeOS processes running on the Chromebook, so , no, basically.

The only way to run a standalone distro on external media on a Chromebook is to set up a dual-boot scenario with `chrx` or similar.

-**DennisLfromGA**

**Reply by wehkah** [reply]
Thanks for explaining!

I must admit: I was expecting a different answer, but that is not your fault! I am desperately searching for a way to turn my Duet into a Linux only machine, but that seems to be not possible (yet). There are Linux distributions out there that can be installed on Chromebooks and that can also be run from usb memory, but those Linux's only run on Chrombooks powered by an x86 cpu, while the Duet uses a ARM.

**Reply by TomTravis** [reply]
you can check the out https://github.com/MrChromebox
you have some work to do.. The bios is write protected

https://github.com/hexdump0815/linux-mainline-on-arm-chromebooks
